server/auth/registrar.go

```diff
@@ -57,9 +57,10 @@ func (auth *AuthManager) handleRegister(conn comms.Link, msg *msgjson.Message) *
 	// Register account and get a fee payment address.
 	feeAddr, err := auth.storage.CreateAccount(acct)
 	if err != nil {
+		log.Debugf("CreateAccount(%v) failed: %v", acct, err)
```

Comment:
Maybe a `log.Infof`?

Reply:
This one can be hit by the user at will (acct already exists in table).  Ideally we'd log differently for legit errors though.